packages/cloudflare/README.md

```diff
@@ -2,8 +2,139 @@
 
 Deploy Next.js apps to Cloudflare!
 
-[OpenNext for Cloudflare](https://opennext.js.org/cloudflare) is an adapter that enables the deployment of Next.js applications to Cloudflare's developer platform.
+[OpenNext for Cloudflare](https://opennext.js.org/cloudflare) is Cloudflare specific adapter that enables deployment of Next.js applications to Cloudflare.
 
 ## Get started
 
-Visit the [OpenNext docs](https://opennext.js.org/cloudflare/get-started) for instructions on starting a new project, or migrating an existing one.
```

Comment:
I think the content in this section might have been added back by accident 😅

Reply:
It's not an accident. 
Having the doc in the same repo & branch makes it easier to update it and to learn about the specifics of that branch.